nuxeo-features/nuxeo-platform-comment/nuxeo-platform-comment/src/main/java/org/nuxeo/ecm/platform/comment/impl/CommentManagerImpl.java

```diff
@@ -267,6 +261,16 @@ private DocumentModel createCommentDocModel(CoreSession mySession, DocumentModel
         return commentDocModel;
     }
 
+    protected DocumentModel createHiddenFolder(CoreSession session, String parentPath, String name) {
+        DocumentModel dm = session.createDocumentModel(parentPath, name, "HiddenFolder");
+        dm.setProperty("dublincore", "title", name);
+        dm.setProperty("dublincore", "description", "");
+        Framework.doPrivileged(() -> dm.setProperty("dublincore", "created", Calendar.getInstance()));
+        DocumentModel parent = session.createDocument(dm); // change variable name to be effectively final
+        setFolderPermissions(parent);
+        return parent;
+    }
+
```

Comment:
Can you explain why this change is not present in master and 10.10?

Reply:
Because tests are not run on this service on master/10.10 as we change the underlying implementation. I will cherry pick it. Thx 👍